mne/viz/_brain/_timeviewer.py

```diff
@@ -939,3 +943,11 @@ def _get_range(brain):
 
 def _normalize(point, shape):
     return (point[0] / shape[1], point[1] / shape[0])
+
+
+def _sanitize(text):
+    from PyQt5.Qt import PYQT_VERSION_STR
+    version = PYQT_VERSION_STR.split('.')
+    if int(version[1]) <= 12:
+        text = text[2:]
```

Comment:
The robust way we do this is `from distutils.version import LooseVersion` and then `if LooseVersion(PYQT_VERSION_STR) < LooseVersion('5.12'):`

Reply:
@larsoner for next time, check here: https://unicode.org/emoji/charts/full-emoji-list.html#light_&_video